spec/ioki/http_adapter_spec.rb

```diff
@@ -15,6 +15,7 @@
       api_client_secret:     SecureRandom.alphanumeric,
       api_client_version:    'api_client_version',
       api_token:             SecureRandom.alphanumeric,
+      api_bleeding_edge:     false,
```

Comment:
I'd actually not have this line here, since do want to test the default, right?

Reply:
Yes! See #215.